docs/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 * :bug:`2004` Show a webui error when JSON-RPC requests fail.
 * :bug:`2039` Return error for negative deposits via REST API
 * :feature:`2011` Add a ``--disable-debug-logfile`` argument to disable the always on debug file if required by the user.
+* :feature:`1950` Breaking change: Better transaction handling on restart. This change breaks binary compatibility with the previous WAL.
```

Comment:
Oh, just saw this. We need to move this to the next release

Reply:
done https://github.com/raiden-network/raiden/pull/2089